### PR TITLE
Adding file upload

### PR DESCRIPTION
**Pull request by AdityaHegde**
- [x] Adding an endpoint to accept form data of a file upload.
- [x] Use the file and call addOrUpdateTableFromFile to import.
- [x] Add support for drag drop on tables section on the left.

### CODE REVIEW

**Review by hamilton — COMMENTED** [review]
This looks good & works great! I'm not going to worry too much about the UX right now since I think the actual user flow should be done in a dedicated PR.

One question: why can't we get this working in dev mode? It seems all we need to do is lift the CORS restriction on the express server to enable port 3000 to request something from port 8080. Is there something else that I'm missing here? We're running 8080 already right?

**Comment by AdityaHegde** [comment]
I felt it will be wasted effort as we will soon be merging sveltekit and socket servers.

**Comment by hamilton** [comment]
Got it ~ we've got some time on the books later & we can talk about that to see what's feasible.

**Comment by hamilton** [comment]
Hmm, this will need to be rebased.

File: src/lib/util/tableFileUpload.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export function uploadFile(file: File, url: string) {
+    const formData = new FormData();
+    formData.append("file", file);
+
+    fetch(url, {
+        method: "POST",
+        body: formData
+    })
+        .then((...args) => console.log(...args))
+        .catch((...args) => console.log(...args));
```
Comment (hamilton) [inline src/lib/util/tableFileUpload.ts:21]:
```suggestion
```

Comment (hamilton) [reply src/lib/util/tableFileUpload.ts:21]: we should probably catch this properly but let's do that in the UX PR.

File: src/server/full-server.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import "../moduleAlias";
+import {RootConfig} from "$common/config/RootConfig";
+import {RillDeveloper} from "$common/RillDeveloper";
+import {ExpressServer} from "./ExpressServer";
+import type {SocketNotificationService} from "$common/socket/SocketNotificationService";
+import {ServerConfig} from "$common/config/ServerConfig";
+
+/**
```
Comment (hamilton) [inline src/server/full-server.ts:8]: is this comment still necessary at this point?

**Review by hamilton — APPROVED** [review]
Thanks for the PR ~ left two small suggestions. Let's merge this so we can begin the frontend.

File: src/server/full-server.ts (hunk 2 above)
Comment (AdityaHegde) [reply src/server/full-server.ts:8]: Nope. The file itself is not needed anymore.

**Pull request merged**